src/components/Typography/Typography.stories.mdx

```diff
@@ -0,0 +1,400 @@
+| `font-size`   | `0.875rem` | `--font-size--1`      |
+| `font-weight` | `300`      | `--font-weight-light` |
+| `line-height` | `1.25`     | n/a                   |
+
+> <p className="text--tag" style={{ color: "black" }}>
+>   Tag
+> </p>
+
+| CSS Attribute | Value     | CSS Variable            |
+| ------------- | --------- | ----------------------- |
+| `font-size`   | `0.75rem` | `--font-size--2`        |
+| `font-weight` | `400`     | `--font-weight-regular` |
+| `line-height` | `1.5`     | n/a                     |
+
+> <p className="text--mini" style={{ color: "black" }}>
+>   Mini
+> </p>
+
+| CSS Attribute | Value      | CSS Variable            |
+| ------------- | ---------- | ----------------------- |
+| `font-size`   | `0.625rem` | `--font-size--3`        |
+| `font-weight` | `400`      | `--font-weight-regular` |
+| `line-height` | `1.5`      | n/a                     |
+
+> **DISCLAIMER**  
+> The `Text` component is still under development and not available to consuming applications.
+
+## Font Family
+
+With website accessibility in mind, font choices for the DS were made with an eye toward clarity and readability. Headings and body content use the following font family assignment:
+
+<Preview>
+  <Story name="Font Family">
+    <p>system-ui, sans-serif</p>
+  </Story>
+</Preview>
+
+The following CSS variables are available through the DS.
+
+```
+--typeface-heading: system-ui, sans-serif;
+--typeface-body: system-ui, sans-serif;
+```
+
+> **RECOMMENDATIONS**  
+> When making font selections for project based components outside the DS, sans serif fonts display better on computers and mobile devices. Avoid decorative or overly stylized fonts, as they are often difficult to read even for users without visual impairments or reading Standard Text Default
+
+## Font Colors
+
+All text within a project, unless specifically noted in mockups, should be black on a white background.
+
+<Preview>
+  <Story name="Font Colors">
+    <>
+      <Heading level={3}>Default Heading Color</Heading>
+      <p>
+        This paragraph shows the default body text color. Curabitur blandit
+        tempus porttitor. Praesent commodo cursus magna, vel scelerisque nisl
+        consectetur et. Integer posuere erat a ante venenatis dapibus posuere
+        velit aliquet. Duis mollis, est non commodo luctus, nisi erat porttitor
+        ligula, eget lacinia odio sem nec elit.
+      </p>
+    </>
+  </Story>
+</Preview>
+
+In cases where specific colors must be applied to text elements, the DS provides CSS variables for the full NYPL color palette. Refer to the [Colors](https://nypl.github.io/nypl-design-system/storybook-static/?path=/docs/style-guide-colors--colors-brand) section of the Style Guide for details.
+
+## Font Weight
+
+The DS uses four font weights to render text. The default font-weight is 300.
+
+<Preview>
+  <Story name="Font Weight">
+    <>
+      <p
+        style={{
+          fontSize: "1rem",
```

Comment:
Are these combinations already set in scss class names? Just curious because I don't know (for font weight and font size) what the appropriate way to change it in a consuming app would be.

Reply:
I rewrote the code for the examples to utilize the actual CSS vars.  Not sure why I didn't do that in the first place.  This will not give one example for how to use the CSS vars to override the default styles.